### ISSUE TITLE
Aria Labels missing from mobile nav toggle

### ISSUE THREAD

**Issue by ecupaio** [Office-of-Digital-Services/California-State-Web-Template-HTML#25]
There should be Aria Labels for opening and closing the mobile navigation on the menu hamburger. 

I can add aria-label="Opens the navigation menu" to the site header component hamburger, but I cannot add aria-label="Closes the navigation menu" to the close toggle due to it being built with JS. The desired output is: 

```
<button 
  type="button"
  class="mobile-control toggle-menu"
  aria-expanded="true"
  aria-controls="navigation"
  aria-label="Closes the navigation menu"
>
  <span></span>
  <span></span>
  <span></span>
  <span></span>
  <span class="sr-only">Menu</span>
</button>

```

Ideally you could add the following code below [this line](https://github.com/Office-of-Digital-Services/California-State-Web-Template-HTML/blob/23736598ace6aa03649c908dfd460d311fdc6579/ca_state_template/js/cagov.core.js#L8705): 
```
navMobileMenuToggleBtn.setAttribute("aria-label", "Closes the navigation menu");
```

**Comment by kkoryaka** [comment]
@ecupaio we will add it to our next release (v6.5.2), thank you for your contribution!

**Comment by ecupaio** [comment]
thanks @kkoryaka. Is there an official way to contribute to the template or was this the best method? I'll have a few more PRs in store as I respond to requests from the department I'm working for.

**Comment by kkoryaka** [comment]
@ecupaio We evaluated your suggestion and attempted implementation; however, we identified that ARIA labels are not inherently translatable, which poses a limitation for multilingual site deployments. As a result, we opted not to include this change in our latest release. If you have further enhancements, please consider submitting pull requests to our state template development repository: [https://github.com/Office-of-Digital-Services/California-State-Web-Template-Development](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-browser/workbench/workbench.html)

**Comment by ecupaio** [comment]
Got it. So why not then just use the sr only span to fulfill accessibility requirements for buttons to have descriptive text and the requirement for page text to be translated?  Does "Menu" suffice? 

Also, thank you for direction on where to add PRs.
